tldw_Server_API/app/core/Chat/chat_service.py

```
@@ -1093,6 +1095,46 @@ def resolve_provider_and_model(
     raw_api_provider = getattr(request_data, "api_provider", None)
     provider_explicit = bool(str(raw_api_provider or "").strip())
 
+    if routing_decision is not None and routing_decision.canonical:
+        try:
+            setattr(request_data, "api_provider", routing_decision.provider)
+        except _CHAT_NONCRITICAL_EXCEPTIONS:
+            pass
```

Ruff (0.15.6)

[warning] Use `contextlib.suppress(_CHAT_NONCRITICAL_EXCEPTIONS)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(_CHAT_NONCRITICAL_EXCEPTIONS): ...`

(SIM105)